pkg/user/tx_client.go

```diff
@@ -40,6 +40,7 @@ import (
 const (
 	DefaultPollTime          = 3 * time.Second
 	txTrackerPruningInterval = 10 * time.Minute
+	evictionPollTime         = 1 * time.Minute
```

Comment:
can we add a quick comment here for what this polling value actually means and how it differs from the other poll time?